indy_node/test/state_proof/test_state_proof_for_get_requests.py

```diff
@@ -176,3 +186,105 @@ def test_state_proof_returned_for_get_claim_def(looper,
     assert data == expected_data
     assert result[TXN_TIME]
     check_valid_proof(result)
+
+
+def test_state_proof_returned_for_get_revoc_reg_def(looper,
+                                                    nodeSetWithOneNodeResponding,
+                                                    sdk_wallet_steward,
+                                                    sdk_pool_handle,
+                                                    sdk_wallet_client,
+                                                    send_revoc_reg_def):
+    revoc_reg_def_data = send_revoc_reg_def[0]['operation']
+
+    req = looper.loop.run_until_complete(build_get_revoc_reg_def_request(
+        sdk_wallet_client[1], get_revoc_reg_def_id(sdk_wallet_steward[1], send_revoc_reg_def[0])))
+    rep = sdk_get_and_check_replies(looper, [sdk_sign_and_submit_req(sdk_pool_handle, sdk_wallet_client, req)])
+    result = rep[0][1]['result']
+
+    expected_data = revoc_reg_def_data
+    del expected_data['type']
+    assert DATA in result
+    data = result.get(DATA)
+    assert data
+    assert data == expected_data
+    assert result[TXN_TIME]
+    check_valid_proof(result)
+
+
+def test_state_proof_returned_for_get_revoc_reg_entry(looper,
+                                                      nodeSetWithOneNodeResponding,
+                                                      sdk_wallet_steward,
+                                                      sdk_pool_handle,
+                                                      sdk_wallet_client,
+                                                      send_revoc_reg_entry):
+    revoc_reg_def = send_revoc_reg_entry[0]
+    revoc_reg_entry_data = send_revoc_reg_entry[1][0]['operation']
+
+    timestamp = int(time.time())
```

Comment:
Do we have a test where `timestamp ` is before the first write of RevocRegEntry?
Do we have a test where `timestamp ` is before the first write to the ledger?

Reply:
fixed

Reply:
Is it really fixed?